__tests__/data/hs_router_input.json

```diff
@@ -161,11 +161,11 @@
           "properties": [
             {
               "property": "email",
-              "value": "testhubspot2@email.com"
+              "value": "testhubspot3@email.com"
             },
             {
               "property": "firstname",
-              "value": "Test Hubspot"
+              "value": "Test Hubspot3"
             }
           ]
         },
```

Comment:
we need to add `"JSON_ARRAY": {},` below ?

Reply:
@utsabc  @akashrpo  how are the tests passing if its missing?

Reply:
It is in the *_input.json part and we are missing the new structure here and while extraction we are extracting from JSON.properties hence, getting the desired result ?